tools/runtime-codegen/Cargo.toml

```diff
@@ -4,15 +4,16 @@ version = "0.1.0"
 description = "Tool for generating bridge runtime code from metadata"
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2021"
-license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+license = "GPL-3.0-only"
```

Comment:
Should we change to "GPL-3.0-or-later WITH Classpath-exception-2.0"?

Reply:
Sorry, I mean - leave the previous version : "GPL-3.0-or-later WITH Classpath-exception-2.0". The whole license stuff is explained here: https://substrate.stackexchange.com/questions/594/how-should-we-choose-a-license - Apache for pallets and other runtime stuff and GPL with classpath for client (also relay). I may be wrong, though - just think that we don't want to restrict license on already available code.

Reply:
Ok, thanks for the explanation ! Reverted the licenses to the previous version.